Cargo.toml

```diff
@@ -14,7 +14,7 @@ getopts-like option parsing.
 categories = ["command-line-interface"]
 
 [dependencies]
-unicode-width = "0.1.5"
+unicode-width = "0.2.0"
```

Comment:
The unicode-width version bump changes the behavior in the following case:
* Treat \n as width 1

Just perusing the code I don't think this crate is affected by that change, because usage within this crate
is within a `for line in desc().trim().lines() {` loop, but a more certain and focused review
for why this crate is unaffected by the change would not be amiss.